### PR TITLE
Add maker-bundle recipe

### PR DESCRIPTION
**Pull request by fabpot**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

**Comment by fabpot** [comment]
@symfony-flex-server review again please

**Comment by fabpot** [comment]
@symfony-flex-server review one more time please

**Comment by fabpot** [comment]
@symfony-flex-server review as I'm still sleeping a bit, messed up last time, sorry.

File: symfony/maker-bundle/1.0/manifest.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "bundles": {
+        "Symfony\\Bundle\\MakerBundle\\MakerBundle": ["dev"]
+    },
+    "aliases": ["maker", "generator"]
```
Comment (xabbuh) [inline symfony/maker-bundle/1.0/manifest.json]: What about also adding `code-generator` (as used in the keywords of the bundle) and maybe also `generator-bundle`?

Comment (fabpot) [reply symfony/maker-bundle/1.0/manifest.json]: Aliases should be short and sweet.

Comment (fabpot) [reply symfony/maker-bundle/1.0/manifest.json]: generator-bundle would be confusing as the current one generates CRUD, not this one. So, make-bundle is still not a replacement for SensioGeneratorBundle (unfortunately).

**Pull request merged**